docs/configuration/unit_testing_rules.md

```diff
@@ -119,6 +119,8 @@ series: <string>
 #         Observation counts in negative buckets. Each represents an absolute count.
 #     - n_offset (int):
 #         The starting index of the first entry in the negative buckets.
+#     - counter_reset_hint (one of 'unknown', 'counter', 'not_counter' or 'gauge')
```

Comment:
I think it's slightly better to have these:
```suggestion
#     - counter_reset_hint (one of 'unknown', 'reset', 'not_reset' or 'gauge')
```

When a new person reads the test, I think "count" would be a little bit ambiguous especially next to gauge.

note: I don't know of plans to add more options, but technically the most precise would be:
```suggestion
#     - counter_reset_hint (one of 'unknown_counter_reset', 'counter_reset', 'not_counter_reset' or 'gauge_type')
```
but that's just too much for now.

Reply:
Done, fixed in 45496e9